### ISSUE TITLE
Add examples.

### ISSUE THREAD

**Issue by ibrahimAlii** [xkcoding/spring-boot-demo#21]
Should we consider add example to each topics?
How to use it and best practice to use it!

**Comment by xkcoding** [comment]
Do you mean to add the corresponding English README translation under each topic?

**Comment by ibrahimAlii** [comment]
Not exactly, In the same README we add example of how to use demo and best practice for it, i.e for `helloworld` demo example would be `http://localhost:8080/demo/hello?who=ibrahim`

**Comment by xkcoding** [comment]
I understand what you mean, in fact, I also do this in the Chinese README under each topic.

I will translate it if I take the time.

If you are willing to help, it would be better. :bowtie:
